pkg/providers/native/native.go

```diff
@@ -306,34 +301,6 @@ func (p *Native) assembleNodeStatus(ssh *types.SSH) (*types.Cluster, error) {
 }
 
 func (p *Native) syncNodes() error {
```

Comment:
useless function?

Reply:
No, the function used in https://github.com/cnrancher/autok3s/pull/421/files#diff-170712b1ae59bad0c19ddb8eb0986fa8a548f242c367f17003470dd62e4ebd24R102